api/src/main/java/uk/gov/dhsc/htbhf/hmrc/controller/HMRCEligibilityController.java

```diff
@@ -0,0 +1,39 @@
+package uk.gov.dhsc.htbhf.hmrc.controller;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.dhsc.htbhf.hmrc.model.EligibilityRequest;
+import uk.gov.dhsc.htbhf.hmrc.model.EligibilityResponse;
+import uk.gov.dhsc.htbhf.hmrc.service.EligibilityService;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("v1/hmrc/eligibility")
+@AllArgsConstructor
+@Slf4j
+@Api(description = "Endpoints for dealing with HMRC Eligibility requests.")
+public class HMRCEligibilityController {
+
+    private final EligibilityService eligibilityService;
+
+    @PostMapping
+    @ApiOperation("Retrieve the eligibility of a person for Child Tax Credits")
```

Comment:
This is actually retrieving their eligibility for Healthy Start, based on HMRC's opinion of their income and the fact that they're getting CTC